src/CONTRIBUTING.md

```diff
@@ -0,0 +1,139 @@
+## Table of Contents
```

Comment:
Did you mean to include `CONTRIBUTING.md` in this PR?

Reply:
No - i must have accidentally added it when switching between branches - thanks!